Vagrantfile

```diff
@@ -0,0 +1,19 @@
+# Encoding: utf-8
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+ENV['VAGRANT_DEFAULT_PROVIDER'] = 'virtualbox'
+
+# http://docs.vagrantup.com/v2/
+Vagrant.configure('2') do |config|
+  config.vm.box = 'ubuntu/trusty64'
+  config.vm.hostname = 'url-py'
+  config.ssh.forward_agent = true
+
+  config.vm.provider :virtualbox do |vb|
+    vb.customize ["modifyvm", :id, "--memory", "1024"]
+    vb.customize ["modifyvm", :id, "--cpus", "2"]
```

Comment:
@miroswan [suggested adding](https://seomoz.atlassian.net/browse/ROGER-1158):
```rb
vb.customize ["modifyvm", :id, "--natdnshostresolver1", "on"]
```
to ensure correct work of VPN. I did not have a chance to test it, but I used to have problems running VPN from home.